pom.xml

```diff
@@ -24,12 +24,36 @@
             <version>${version.keycloak}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
These dependencies are not needed. Also, the Jakarta Sodium dependencies can be safely removed.